README.md

```diff
@@ -228,236 +250,52 @@ messages about the conversion.
-for databases larger than a couple of GB.
+for databases larger than a couple of GB. Please refer below mentioned links for details on
+data conversion of PostgreSQL and MySQL database:
 
-### Timestamps and Timezones
-
-As noted earlier when discussing [schema conversion of
-TIMESTAMP](#timestamp), there are some sutble differences in how timestamps are
-handled in PostgreSQL and Spanner.
-
-During data conversion, PostgreSQL `TIMESTAMPTZ` values are converted to UTC and
-stored in Spanner. The conversion proceeds as follows. If the value has a
-timezone, that timezone is respected during the conversion to UTC. If the value
-does not have a timezone, then we look for any `set timezone` statements in the
-pg_dump output and use the timezone specified. Otherwise, we use the `TZ`
-environment variable as the timezone, and failing that, we use the local system
-timezone default (as determined by Go).
-
-In constrast, conversion of PostgreSQL `TIMESTAMP` values proceeds by ignoring
-any timezone information and just treating the value as UTC and storing it in
-Spanner.
-
-### Strings, character set support and UTF-8
-
-Spanner requires that `STRING` values be UTF-8 encoded. All Spanner functions
-and operators that act on `STRING` values operate on Unicode characters rather
-than bytes. Since we map many PostgreSQL types (including `TEXT` and `CHAR`
-types) to Spanner's `STRING` type, HarbourBridge is effectively a UTF-8 based
-tool.
-
-Note that the tool itself does not do any encoding/decoding or UTF-8 checks: it
-passes through data from pg_dump to Spanner. Internally, we use Go's string
-type, which supports UTF-8.
+- [PostgreSQL data conversion](postgres/README.md#data-conversion)
+- [MySQL data conversion](mysql/README.md#data-conversion)
 
 ## Troubleshooting Guide
 
 The following steps can help diagnose common issues encountered while running
 HarbourBridge.
 
-### 1. Verify pg_dump configuration
+### 1. Verify driver configuration
+
+If you are using the dump approach then check that pg_dump/mysqldump is correctly
+configured to connect to your PostgreSQL/MySQL database. If you are using the information
+schema approach then check that environment variables are correctly configured to connect
+to your PostgreSQL/MySQL database. Also note the below mentioned points:
 
-First, check that pg_dump is correctly configured to connect to your PostgreSQL
-database. Note that pg_dump uses the same options as psql to connect to your
-database. See the [psql](https://www.postgresql.org/docs/9.3/app-psql.html) and
-[pg_dump](https://www.postgresql.org/docs/9.3/app-pgdump.html) documentation.
+- **pgdump**: pg_dump uses the same options as psql to connect to your
+  database. See the [psql](https://www.postgresql.org/docs/9.3/app-psql.html) and
+  [pg_dump](https://www.postgresql.org/docs/9.3/app-pgdump.html) documentation.
 
 Access to a PostgreSQL database is typically configured using the following
```

Comment:
Formatting seems broken here: the "Access to a PostgreSQL database ..." is part of the pgdump bullet, but it is not indented?

Same for the other bullets for postgres, mysqldump and mysql.

Reply:
Yeah, adding subsections would be better.